new/rfcXXX-ruby_data_bag_items.md

```diff
@@ -0,0 +1,131 @@
+---
+RFC: unassigned
+Author: Eric Krupnik <ekrupnik@copyright.com>
+Status: Draft
+Type: Standards Track
+---
+# Ruby data bag items
+
+Currently, Chef only supports JSON data bag items, and not Ruby. Chef **does**
+support having other configuration-based files (such as `environments` and `roles`)
+be written in either JSON or Ruby. JSON does not allow comments, so it becomes
```

Comment:
data bags are missing tests, but environments, nodes and roles have tests:

https://github.com/opscode/chef/blob/master/spec/data/object_loader/environments/test.json
https://github.com/opscode/chef/blob/master/spec/data/object_loader/nodes/test.json
https://github.com/opscode/chef/blob/master/spec/data/object_loader/roles/test.json